packages/hoppscotch-common/src/components/importExport/Base.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <HoppSmartModal
+    dialog
+    :title="t(modalTitle)"
+    styles="sm:max-w-md"
```

Comment:
```suggestion
    :title="t(modalTitle)"
    styles="sm:max-w-md"
    dialog
```

The [style guide](https://eslint.vuejs.org/rules/attributes-order.html#rule-details) recommends to keep boolean props below static and dynamic props.